courses/admin.py

```diff
@@ -85,6 +89,10 @@ class CourseRunEnrollmentAdmin(admin.ModelAdmin):
     """Admin for CourseRunEnrollment"""
 
     model = CourseRunEnrollment
+    search_fields = ["user", "company", "CourseRun"]
+    list_filter = ["active", "change_status"]
+    list_fields = ["user.email", "change_status"]
```

Comment:
What is the intention here? I see `list_display`, `list_display_links`, `list_editable` in `ModelAdmin` but there's no reference to `list_fields` anywhere.